### PR TITLE
Notch8 Changes

### PR DESCRIPTION
**Pull request by sara-g**
Changes from Notch8: 
-Import and Export Places (including exporting Place URLs, Place Categorizations)
-Admin Place Search 
-Add a Place to a Story Workflow

### CODE REVIEW

File: db/migrate/20210705195115_set_default_for_place_status_option.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class SetDefaultForPlaceStatusOption < ActiveRecord::Migration[6.1]
```
Comment (dvcv) [inline db/migrate/20210705195115_set_default_for_place_status_option.rb:1]: Does this file need to be uncommented or removed?

File: app/javascript/js/places.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        data: $(this).sortable('serialize')
+      })
+    }
+  });
+
+  // Select Location After ArcGIS Search and Populate Form
+  $('#select-result').on('click', function() {
+    let selectedText = $('#place-autocomplete').val();
+    $.ajax({
+      url: `https://utility.arcgis.com/usrsvcs/appservices/I9WaUoR33Blxi00f/rest/services/World/GeocodeServer/findAddressCandidates?SingleLine=${selectedText}&outFields=*&f=pjson`,
```
Comment (dvcv) [inline app/javascript/js/places.js:28]: Is it possible to re-use the URL constant stored in the config/initializers/esri.rb file incase the url ever changes.

File: app/javascript/js/places.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+               resultList.innerHTML = "";
+               response["suggestions"].forEach(function(item) {
+                   var option = document.createElement('option');
+                   option.value = item["text"];
+                   option.id = item["magicKey"];
+                   resultList.appendChild(option);
+               });
+           }
+       };
+       window.hinterXHR.open("GET", `https://utility.arcgis.com/usrsvcs/appservices/I9WaUoR33Blxi00f/rest/services/World/GeocodeServer/suggest?text=${input.value}&outFields=*&f=pjson`, true);
```
Comment (dvcv) [inline app/javascript/js/places.js:81]: Same here

File: db/migrate/20210705195115_set_default_for_place_status_option.rb (hunk 1 above)
Comment (sara-g) [reply db/migrate/20210705195115_set_default_for_place_status_option.rb:1]: I think we can delete it, but I commented it out to be safe.

File: app/javascript/js/places.js (hunk 2 above)
Comment (sara-g) [reply app/javascript/js/places.js]: We added a JS variable to this file to use in those 2 places.  Was not sure how to pull it in from esri.rb.

File: app/javascript/js/places.js (hunk 3 above)
Comment (sara-g) [reply app/javascript/js/places.js]: same as above

**Pull request merged**